scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -378,47 +378,6 @@ <h6 class="font-weight-bold">{{ image.title }}</h6>
   </div> <!--  end of card div -->
 {% endmacro %}
 
-{% macro acmg_form(selected=None) %}
```

Comment:
Wow, was this not used?

Reply:
I think not - looks like a copy or merge issue from moving stuff over to that common components file.